core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -551,6 +551,22 @@ package object config {
       .checkValue(_.endsWith(java.io.File.separator), "Path should end with separator.")
       .createOptional
 
+  private[spark] val STORAGE_FALLBACK_STORAGE_NUM_THREADS_FOR_SHUFFLE_READ =
+    ConfigBuilder("spark.storage.fallbackStorage.num.threads.for.shuffle.read")
```

Comment:
Every `.` means a new namespace in Apache Spark configuration namespace scheme. For example, this line introduces 4 new namespaces. Please avoid using `.`.
- spark.storage.fallbackStorage.num.*
- spark.storage.fallbackStorage.num.threads.*
- spark.storage.fallbackStorage.num.threads.for.*
- spark.storage.fallbackStorage.num.threads.for.shuffle.*